saleor/graphql/core/types/common.py

```diff
@@ -77,3 +77,22 @@ def get_adjusted(image, alt, size, rendition_key_set, info):
             url = image.url
         url = info.context.build_absolute_uri(url)
         return Image(url, alt)
+
+
+class PriceRangeInput(graphene.InputObjectType):
+    gte = graphene.Float(
+        description='Price greater than or equal', required=False)
+    lte = graphene.Float(
+        description='Price less than or equal', required=False)
+
+
+class DateRangeInput(graphene.InputObjectType):
+    from_date = graphene.Date(description='Starting date from', required=False)
```

Comment:
I guess we don't have to duplicate the word "date" in the field name since it's included in the type name, and probably it will be understandable from the filter name as well. "From" and "To" should be fine.

Reply:
And for the descriptions maybe simply: "Start date" and "End date".